optimum/exporters/onnx/model_configs.py

```diff
@@ -2629,6 +2629,51 @@ class EncoderDecoderOnnxConfig(EncoderDecoderBaseOnnxConfig):
     DEFAULT_ONNX_OPSET = 14  # uses SDPA in Transformers, hence opset>=14.
 
 
+class TimesFMDummyInputGenerator(DummyInputGenerator):
```

Comment:
could make sense to re-use `DummyPatchTSTInputGenerator` instead (with some extra modifications), let's wait for https://github.com/huggingface/transformers/pull/34082/ to be merged

Reply:
@echarlaix one question i had about the https://github.com/huggingface/transformers/pull/34082  in the timesfm_config i do not think my `TimesFmOnnxConfig` is correct... do you agree?

Reply:
Yes it shouldn't inherit from `OnnxSeq2SeqConfigWithPast` (which should be used for the export of encoder-decoder models). I think you can directly remove `TimesFmOnnxConfig` in https://github.com/huggingface/transformers/pull/34082/ as it shouldn't be used anywhere from what I know (also I'm expecting anything onnx related to be deprecated in v5)

Reply:
ok let me remove it... it would be good to then get it working via transformers.js... let me remove it first and then get back to you!

Reply:
sure thanks, also cc @michaelbenayoun @JingyaHuang @IlyasMoutawwakil who can take a look while I'm ooo